controllers/fio/job_test.go

```diff
@@ -40,7 +41,10 @@ var _ = Describe("fio job", func() {
 					CmdLineArgs: "--name=randwrite --iodepth=1 --rw=randwrite --bs=4m --direct=1 --size=256M --numjobs=1",
 				},
 			}
-			job = NewJob(&cr)
+			configMap := corev1.ConfigMap{
+				ObjectMeta: metav1.ObjectMeta{Name: "cm"},
```

Comment:
Having a fixed 'cm' name will enable only one customConfig'd job per namespace.
We should also include the CR name in the object's name here.

Reply:
pls ignore, just realized that this is just a test